### ISSUE TITLE
React 16

### ISSUE THREAD

**Issue by dmitrykurmanov** [surveyjs/surveyjs#720]
related to https://github.com/surveyjs/surveyjs/issues/718

need to support the latest react 16 version
https://reactjs.org/blog/2017/09/26/react-v16.0.html#breaking-changes

from @mikecousins  ( https://github.com/surveyjs/surveyjs/issues/703 ):

> We still get some warnings when we install packages:
> 
> warning "survey-react@0.12.32" has incorrect peer dependency "react@^15.0.1".
> warning "survey-react@0.12.32" has incorrect peer dependency "react-dom@^15.0.1".
> Maybe the dependency in package.json needs to be updated?

**Comment by mikecousins** [comment]
Is this planned soon? It's a pretty quick and easy fix in package.json.

**Comment by andrewtelnov** [comment]
@dmitrykurmanov @tsv2013 Let's try to use the latest version and see how it works.

Thank you,
Andrew

**Comment by enricribas** [comment]
bump

**Comment by andrewtelnov** [comment]
@dmitrykurmanov @tsv2013 Let's check this issue tommorrow.

Thank you,
Andrew

**Comment by enricribas** [comment]
Input fields cannot be filled out with React 16. They do not update.

**Comment by dmitrykurmanov** [comment]
Hello Enric, sorry for the late response, I can't reproduced the issue. I created an example with inputs and React16 and it looks ok. https://plnkr.co/edit/g7b506tzx7rf8jyqCoh9?p=preview. Could you please check it?

**Comment by enricribas** [comment]
Had a chance to look into it, and it seems that the issue is with Found router, and React 16 and not with surveyjs. Thank you for looking into it. We will have to deal with replacing our router... again. :(

**Comment by enricribas** [comment]
Sorry guys, but I am still having an issue but only with surveyjs fields. I cannot edit them. I thought it was found-router but I replaced that with react-router. Then I thought it was css_modules, so I removed that. 

So now with react-router, react 16, and surveyjs fields are not editable. Other non-surveyjs fields are editable and work fine. So there is some issue with react-router, react 16 and surveyjs
